spec/activerecord-multi-tenant/query_rewriter_spec.rb

```diff
@@ -113,17 +117,6 @@
     let!(:manager1) { Manager.create(name: 'Manager 1', project: project1, account: account) }
     let!(:manager2) { Manager.create(name: 'Manager 2', project: project2, account: account) }
 
-    before(:each) do
-      @queries = []
-      ActiveSupport::Notifications.subscribe('sql.active_record') do |_name, _started, _finished, _unique_id, payload|
-        @queries << payload[:sql]
-      end
-    end
-
-    after(:each) do
-      ActiveSupport::Notifications.unsubscribe('sql.active_record')
-    end
```

Comment:
Refactored some existing tests as they also contained the same kind of unnecessary code that was pointed out in the review.